### ISSUE TITLE
Add a link to github.

### ISSUE THREAD

**Issue by tylervz** [jglamine/calvinwebprint#8]
Perhaps put this on the bottom of the page "fork me on github", or something like that. Make it small and unobtrusive. This issue can get solved once #16 gets solved.

**Comment by tylervz** [comment]
@jglamine, should we only have this link on the main page?

**Comment by jglamine** [comment]
Yah, to https://github.com/jglamine/calvinwebprint.

**Comment by jglamine** [comment]
We should have a consistent footer at the bottom of the page. It can have links to things like settings, github, stats, and an about page. I'd also like to consider getting rid of the help page, and replacing it with a short "about" page.

A footer can be done through a jinja template in flask. Right now the main page is being served as a static file. This should be replaced with a route in flask which renders a template. Templates can be nested, so they can all be surrounded by the header and the footer.

For an example, see here, where the template "extends" the base.html template: https://github.com/jglamine/phage/blob/master/webphamerator/app/templates/database.html

**Comment by tylervz** [comment]
Okay, sounds good.

**Comment by jglamine** [comment]
Perhaps moving things to a temple should be broken out into its own issue. Is it possible to make an issue depend on other issues?

**Comment by tylervz** [comment]
I agree it should be its own issue.

**Comment by tylervz** [comment]
This issue is dependent on #16
